packages/netlify-cms-core/src/lib/formatters.ts

```diff
@@ -166,11 +166,25 @@ export const previewUrlFormatter = (
    * URL path.
    */
   const basePath = trimEnd(baseUrl, '/');
-  const pathTemplate = collection.get('preview_path') as string;
+
+  let filePathTemplate;
+  let fileDateField;
+  if (collection.get('type') === 'file_based_collection') {
```

Comment:
You can use the `const` value from:
https://github.com/netlify/netlify-cms/blob/2146b3605f337e808f0e54c92cd2b601296f23a6/packages/netlify-cms-core/src/constants/collectionTypes.js#L1